src/Hatchinq/Divider.elm

```diff
@@ -26,11 +30,14 @@ type alias Config =
 
 type alias InternalConfig =
     { color : Color
+    , height : Length
```

Comment:
these configurations shouldn't be here. There are already a part of the Attribute